core/src/main/java/org/helsinki/paytrail/model/payments/PaytrailPaymentResponse.java

```diff
@@ -0,0 +1,22 @@
+package org.helsinki.paytrail.model.payments;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import java.io.Serializable;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+@Data
+@Accessors(chain = true)
+public class PaytrailPaymentResponse implements Serializable {
+    private String transactionId;
+    private String href;
+    private String terms;
+    private List<PaymentMethodGroupData> groups;
+    private List<Provider> providers;
+    private String reference;
+    private JsonNode customProviders;
```

Comment:
Could this be some kind of dto?

Reply:
It is a DTO like all these other classes under model/payments. Maybe I could just rename the class to PaytrailPaymentTransaction.